### PR TITLE
Documentation for MDL-80259

### PR DESCRIPTION
**Pull request by roland04**

### CODE REVIEW

File: docs/devupdate.md [hunk 1]
```diff
@@ -40,6 +40,30 @@ will only update the second enrol instance.
 
 A new method `enrol_plugin:find_instance()` is added to the enrol plugin interface to allow plugins to find an existing instance of the same type in the course. If you want your enrolment method to be supported in CSV course upload, you need to implement this [method](./apis/plugintypes/enrol#enrol_pluginfind_instance-stdclass)
 
+## Bootstrap preparations for version 5
+
+<Since version="4.4" issueNumber="MDL-71979" />
+
+Some of the Bootstrap 4 classes will deprecated or dropped in its version 5. To prepare for this, some of the current Bootstrap 4 classes usages have been replaced with version 5 compatible classes. This will help us to upgrade to Bootstrap 5 in the future.
+
+- Badge color class helpers `.badge-[warning|info|...]` have been replaced with `.bg-warning` `.bg-info`, ... and the corresponding text color classes `.text-dark` `.text-white` to meet accessibility contrast.
```
Comment (andrewnicols) [inline docs/devupdate.md]:
```suggestion
- Badge colour class helpers `.badge-[warning|info|...]` have been replaced with `.bg-warning` `.bg-info`, ... and the corresponding text color classes `.text-dark` `.text-white` to meet accessibility contrast.
```

We use Australian English spelling :)

Comment (andrewnicols) [inline docs/devupdate.md]: This whole section is confusing. Shoudl we be using `bg-warning` with `text-dark`?

File: docs/devupdate.md [hunk 2]
```diff
@@ -40,6 +40,30 @@ will only update the second enrol instance.
...
+- Also `.badge-pill` has been replaced with `.rounded-pill`
```
Comment (andrewnicols) [inline docs/devupdate.md:50]:
```suggestion
- The `.badge-pill` class has been replaced with `.rounded-pill`
```

**Review by andrewnicols — CHANGES_REQUESTED** [review]
From the changes exemplified here, this seems like a detrimental change to be making. Why are we making it?

File: docs/devupdate.md (hunk 1 above)
Comment (roland04) [reply docs/devupdate.md]: Ouch! Too much time reading Bootstrap documentation. Please don't tell @paulholden about it :sweat_smile:

Comment (roland04) [reply docs/devupdate.md]: You are totally right, it was super-confusing. I've re-written it

**Comment by roland04** [comment]
The origin of these changes is the Bootstrap 5 upgrade. According to [the Bootstrap migration guide](https://getbootstrap.com/docs/5.0/migration/#badges) these badge classes will be dropped in version 5

**Comment by andrewnicols** [comment]
Converting to draft to prevent accidental merge as the linked issues are not ready

File: docs/devupdate.md (hunk 1 above)
Comment (roland04) [reply docs/devupdate.md]: The new version was based the in the BS5 migration documentation https://getbootstrap.com/docs/5.0/migration/#badges

**Comment by aanabit** [comment]
These changes seems OK to me and are ready to merge imho. They are waiting for MDL-80259 to be integrated, so I won't merge the PR for now.

**Pull request merged**